Taboola SDK Swift Sample/Taboola SDK Swift Sample/SDK Standard and JS/TaboolaSTDandJSCollectionView.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  TaboolaSTDandJSCollectionView.swift
+//  Taboola API Swift Sample
+//
+//  Created by Liad Elidan on 17/07/2019.
+//  Copyright © 2019 LiadElidan. All rights reserved.
+//
+
+import UIKit
+import TaboolaSDK
+import WebKit
+
+// Dedup time-stamp
+var viewId: String = {
```

Comment:
every time you use the viewID you set it again. every time it will get a different value and its not what we need.

Reply:
fixing